client/k8s/application/Application.go

```diff
@@ -65,15 +65,23 @@ type EventsResponse struct {
 	Events *apiv1.EventList `json:"events,omitempty"`
 }
 
+func contains(slice []string, item string) bool {
+	set := make(map[string]struct{}, len(slice))
```

Comment:
Isn't it a better solution?

for _, singleItem := range slice {
		if singleItem == item {
			return true
		}
	}
return false

Reply:
done, logical changes applied